workspaces/ui-v2/src/lib/new-regions-interpreter.ts

```diff
@@ -36,19 +36,22 @@ export async function newRegionInterpreters(
   }
 }
 
+// TODO QPB - move IInterpretation generation into ParsedDiff
 function newContentType(
   udiff: ParsedDiff,
   location: IParsedLocation,
   learnedBodies: ILearnedBodies
 ): IInterpretation {
   if (udiff.isA(DiffTypes.UnmatchedRequestBodyContentType)) {
+    const contentType =
+      location.data.type === 'request' ? location.data.contentType : '';
     const learnedRequestBody = learnedBodies.requests.find(
```

Comment:
This is a good example of why the ParsedDiff class is a bad abstraction - location is _from_ ParsedDiff - but to make this function correct, we need to check both the location and the diff type